integration/sbom_test.go

```diff
@@ -10,7 +10,6 @@ import (
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 
-	ftypes "github.com/aquasecurity/trivy/pkg/fanal/types"
```

Comment:
mistake?

Reply:
hmm, interesting I am not aware of it ... let me check and fix it.